spec.emu

```diff
@@ -1080,7 +1079,10 @@ contributors: Nicolò Ribaudo
               1. <ins>For each ModuleRequest Record _required_ of _module_.[[RequestedModules]], do</ins>
                 1. <ins>Let _requiredModule_ be GetImportedModule(_module_, _required_.[[Specifier]]).</ins>
                 1. <ins>If _required_.[[Phase]] is ~defer~, then</ins>
-                  1. <ins>Perform GatherAsynchronousTransitiveDependencies(_requiredModule_, _evaluationList_).</ins>
+                  1. <ins>Let _additionalModules_ be GatherAsynchronousTransitiveDependencies(_requiredModule_).</ins>
+                  1. <ins>For each Module Record _additionalModule_ of _additionalModules_, do</ins>
+                    1. <ins>If _additionalModule_ is not in _evaluationList_, then</ins>
+                      1. <ins>Append _additionalModule_ to _evaluationList_.</ins>
```

Comment:
I don't think you need to go through all this to describe what you're really trying to do.

```suggestion
                  1. <ins>Remove any duplicate entries from _additionalModules_.</ins>
```

We talk about Lists having duplicate entries all over the place.

Reply:
Does this preserve ordering? I need to ensure that « x » + « y, x » becomes « x, y » and not « y, x »

Reply:
Ah, good point. No I don't think it would.